olp-cpp-sdk-core/src/cache/DefaultCacheImpl.cpp

```diff
@@ -836,13 +853,13 @@ DefaultCache::StorageOpenResult DefaultCacheImpl::SetupMutableCache() {
   auto status = mutable_cache_->Open(settings_.disk_path_mutable.get(),
                                      settings_.disk_path_mutable.get(),
                                      storage_settings, OpenOptions::Default);
-  if (status == OpenResult::Fail) {
+  if (status == OpenResult::Fail || status == OpenResult::Corrupted) {
```

Comment:
I don't think we need this for the mutable cache. I would though add a log in case we repaired it and there is a `lost/` folder present so that users may know that some data was lost. Maybe also delete the lost folder as this is extra disk space that is used.